pkg/config/agent/config.go

```diff
@@ -182,9 +182,8 @@ type AgentConfig struct {
 	// 2. TransportInterfaceCIDRs
 	// 3. The Node IP
 	TransportInterfaceCIDRs []string `yaml:"transportInterfaceCIDRs,omitempty"`
-	// The names of the interfaces on Nodes that are used to forward multicast traffic.
-	// Defaults to transport interface if not set.
-	MulticastInterfaces []string `yaml:"multicastInterfaces,omitempty"`
+	// Multicast configuration options.
+	Multicast MulticastConfig `yaml:"multicast,omitempty"`
```

Comment:
Since multicast is alpha, I'm ok with changing the configuration directly without backwards compatibility support, but want to check if @jianjuns @antoninbas is concerned.

Reply:
Same